packages/cms-lib/hubdb.js

```diff
@@ -141,6 +141,10 @@ function convertToJSON(table, rows) {
 }
 
 async function downloadHubDbTable(portalId, tableId, dest) {
+  if (fs.pathExistsSync(dest)) {
```

Comment:
How do we want to handle the situation where `dest` is not an existing file?

Reply:
Currently if `dest` isn't an existing path, it will get created. Are you thinking that we should still validate that `dest` has a `.json` extension?

Reply:
Yeah, I think so.